Examples/proj/Program.cs

```diff
@@ -110,6 +136,85 @@ public static string NullDeReferenceBad()
     {
         return null;
     }
+    public static void TestNullDerefExpectError()
```

Comment:
When I run infer on this example, no null dereference was reported. I was running the latest Infer main branch